### ISSUE TITLE
Incoming data format

### ISSUE THREAD

**Issue by ghost** [tanelih/phonegap-bluetooth-plugin#17]
Hello.
I have been doing some research about this issue. I am doing readings with an RFID reader and some tags. 

Offtopic but related (?): There is a strange behaviour with the plugin. When i start connection and run the startConnectionManager method and then i perform some readings, the plugin notifies on 2 incoming streams. The first one seems to be a random partial of the full stream. And the second stream most of the times becomes the full stream. I.e:
1. First reading attempt from a tag:
 a. First incoming data: ":"
 b. Second incoming data: ":FXDB=001553930C123019233192"
2. Secont reading attempt from a tag:
 a. First incoming data: ":F"
 b. Second incoming data: "FXDB=001553930C123019233192"

I figured this out with a method like this:

bluetooth.startConnectionManager(incoming_data,fail);

function incoming_data(data){
    data = $.trim(data);
    if(data.length < 5){ //random number, any incoming data with less than 5 chars is not valid for me
          console.log("Ignoring this incoming data: " + data);
          return;
    }
    data  = data.split("=");
    data = data[1] //Now i get just the number, this is worthful data for me
    process_data(data);
}

Now, turns out that the real data (what is suppose to be read with the reader and be received by the application) from my tag is printed on a side of the it. i.e: 0000123
My stick reader has a led screen that shows the same number printed in the tag and this tells me that somehow the incoming data is parsed to this number.

Summing up, the plugin is receiving something like: "FXDB=13C0000C0150180000000" and what must be received  is something like: "0003091"

I was looking in the BluetoothPlugin.java (line 953) at BluetoothWrapper.MSG_READ case and saw that this is where data is parsed and sent back to the plugin. I have tried to change the format on the constructor of the String 'data' object with no luck, also i have played around this case but im clueless now.
